fix checkpoint/restore tests on actuated-arm64

kolyshkin opened the pull request:
While investigating #4274 I found a few issues and rough edges with our c/r tests.

This, together with https://github.com/checkpoint-restore/criu/pull/2403 (which is already [applied to CRIU package](https://github.com/opencontainers/runc/pull/4276#issuecomment-2104283949) we use) fixes CI failures we see now.
 
 Fixes: #4274.

kolyshkin commented:
@adrianreber @rst0git PTAL

alexellis commented:
Hi @kolyshkin we can enable CONFIG_NF_TABLES_INET for you. What else is needed or dependent?

rst0git commented:
@kolyshkin I've updated the CRIU packages in [OBS](https://build.opensuse.org/package/show/devel:tools:criu/criu) with the patch from https://github.com/checkpoint-restore/criu/pull/2403.

kolyshkin commented:
> @kolyshkin I've updated the CRIU packages in [OBS](https://build.opensuse.org/package/show/devel:tools:criu/criu) with the patch from [checkpoint-restore/criu#2403](https://github.com/checkpoint-restore/criu/pull/2403).

Awesome; I've removed the last commit so we're now running all tests on ARM (except, of course, pre-dump).

rst0git reviewed (commented):
LGTM

lifubang reviewed (approved):
LGTM, please update your PR description.
